DevEdu.API/DevEdu.API/Models/OutputModels/UserExistingFullInfoOutPutModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using DevEdu.DAL.Enums;
+
+namespace DevEdu.API.Models.OutputModels
+{
+    public class UserExistingFullInfoOutPutModel : UserInfoOutPutModel
```

Comment:
why have you renamed UserFullInfoOutPutModel?

Reply:
I was making another model for testing